ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,199 @@
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    @objc public var avatarImage: UIImage? {
+        didSet {
+            if let subtitleLabel = subtitleLabel {
+                setupAvatarView(with: titleLabel.text ?? "", and: subtitleLabel.text ?? "")
+            } else {
+                setupAvatarView(with: titleLabel.text ?? "")
+            }
+        }
+    }
+
+    private let avatarView: AvatarView
+    private var titleLabel: UILabel
+    private var subtitleLabel: UILabel?
+    private var labelContainer: UIView
+
+    /// Initializes the contact view by creating an avatar view with a first name and last name
+    ///
+    /// - Parameters:
+    ///   - title: String that will be the text of the top label
+    ///   - subtitle: String that will be the text of the bottom label
+    @objc public convenience init(title: String, subtitle: String) {
+        self.init(title: title, subtitle: subtitle, identifier: nil)
+    }
+
+    /// Initializes the contact view by creating an avatar view with an identifier
+    ///
+    /// - Parameters:
+    ///   - identifier: String that will be used to identify the contact (e.g. email, phone number, first name)
+    @objc public convenience init(identifier: String) {
+        self.init(title: nil, subtitle: nil, identifier: identifier)
+    }
+
+    private init(title: String?, subtitle: String?, identifier: String?) {
+        avatarView = AvatarView(avatarSize: .extraExtraLarge, withBorder: false, style: .circle)
+        labelContainer = UIView(frame: .zero)
+        titleLabel = UILabel(frame: .zero)
+        super.init(frame: .zero)
+
+        if let title = title, let subtitle = subtitle {
+            setupAvatarView(with: title, and: subtitle)
+            setupTitleLabel(using: title)
+            setupSubtitleLabel(using: subtitle)
+        } else if let identifier = identifier {
+            setupAvatarView(with: identifier)
+            setupTitleLabel(using: identifier)
+        }
+
+        backgroundColor = Colors.surfacePrimary
+        setupLayout()
+    }
+
+    public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private func setupAvatarView(with firstName: String, and lastName: String) {
+        let fullName = firstName + " " + lastName
+        avatarView.setup(primaryText: fullName, secondaryText: nil, image: avatarImage)
+    }
+
+    private func setupAvatarView(with identifier: String) {
+        avatarView.setup(primaryText: identifier, secondaryText: nil, image: avatarImage)
+    }
+
+    private func setupLayout() {
+        var constraints = [NSLayoutConstraint]()
+        constraints.append(widthAnchor.constraint(equalToConstant: Constants.contactWidth))
+        constraints.append(contentsOf: avatarLayoutConstraints())
+
+        avatarView.translatesAutoresizingMaskIntoConstraints = false
+        labelContainer.addSubview(avatarView)
```

Comment:
Not really a "labelContainer" if it contains the avatar view. Is this intentional? If so, we probably need a better name. Also, why is this container view needed if it contains all the views that our root view contains?

Reply:
Oh this was not intentional. That was a mistake and should have only contained `titleLabel` and `subtitleLabel`. I'll fix that in a new commit.